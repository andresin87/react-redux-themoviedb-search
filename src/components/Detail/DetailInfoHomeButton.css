.DetailInfoHomeButton {
  position: absolute;
  right: 15px;
  top: 90px;
  background-color: #555;
  border-color: #555;
  color: white;
  font-size: 16px;
  padding: 10px 24px;
  border: none;
  cursor: pointer;
  border-radius: 5px;
  height: 40px;
  text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.12);
  -webkit-box-shadow: 0 2px 0 rgba(0, 0, 0, 0.035);
  box-shadow: 0 2px 0 rgba(0, 0, 0, 0.035);
}
