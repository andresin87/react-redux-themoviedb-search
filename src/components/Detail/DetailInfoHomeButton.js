import React, { PureComponent } from 'react';
import { NavLink } from 'react-router-dom';

import './DetailInfoHomeButton.css';

class DetailInfoHomeButton extends PureComponent {
  render() {
    return (
      <NavLink
        to="/"
        activeClassName="DetailInfoHomeButton"
        style={{ color: 'white' }}
      >
        Home
      </NavLink>
    );
  }
}

export default DetailInfoHomeButton;
