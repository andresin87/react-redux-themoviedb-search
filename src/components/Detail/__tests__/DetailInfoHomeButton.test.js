import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import DetailInfoHomeButton from '../DetailInfoHomeButton';

describe('DetailInfoHomeButton component', () => {
  const component = (
    <Router>
      <DetailInfoHomeButton />
    </Router>
  );

  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(component, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('Snapshot matchs', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
