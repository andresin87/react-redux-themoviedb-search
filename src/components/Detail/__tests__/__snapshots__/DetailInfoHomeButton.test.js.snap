// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailInfoHomeButton component Snapshot matchs 1`] = `
<a
  aria-current="page"
  className="DetailInfoHomeButton"
  href="/"
  onClick={[Function]}
  style={
    Object {
      "color": "white",
    }
  }
>
  Home
</a>
`;
